src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,9 +8,14 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String EXAMPLE = "a/123, Clementi Ave 3, #12-34, 231534";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses to be in this format";
+    public static final String ADDRESS_VALIDATION_REGEX = ",+";
+    
+    private final Block block;
```

Comment:
Why is this final? Do you have setter methods to change these variables, if a value change? Check out the implications of making a variable final